eng/versioning/version_client.txt

```diff
@@ -61,8 +61,8 @@ com.microsoft.azure:azure-spring-boot-metrics-starter;2.2.4;2.2.5-beta.1
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 
-unreleased_com.azure:azure-core;1.6.0-beta.1
-unreleased_com.azure:azure-core-test;1.3.0-beta.1
+unreleased_com.azure:azure-core;1.6.0
```

Comment:
Do we remove this every time bump the release version?

Reply:
Yes, the `unreleased` version needs to match `current` version. After the modules are shipped these tags are removed.

Reply:
@sima-zhu the unreleased_ entries need to be removed after the library has been released. Unreleased entries should be one of the only times that an auto increment PR needs to be modified, after the fact, before it can be merged.